Kanban-Process-for-Issue-Tracking.md

```diff
@@ -1,29 +1,53 @@
+
+## TriBITS GitHub Issue Process
+
+The intended Kanban process lifecycle for a **TriBITS GitHub Issue** is as follows (consistent with [TriBITS/CONTRIBUTING.md](https://github.com/TriBITSPub/TriBITS/blob/master/CONTRIBUTING.md)):
 
-The intended Kanban process lifecycle is as follows:
 * Someone creates a new Issue as a suggestion (and it automatically is placed in the Backlog).
 * The Product Owner then takes the Issue and reviews it in the Backlog and comments on it (or closes it if justified).
-* Regularly Issues are moved into "ToDo" or "Selected" when they are high enough priority and are ready to be worked.  (We should not cram a bunch of stuff into "Selected".  We should keep that list short so that it is clear what to work on next.)
-* Where adequate capacity is freed up, an Issue is moved from "Selected" to "In Progress" by the developer and it is actively worked.
+* Issues are regularly are moved into "ToDo" or "Selected" when they are high enough priority and are ready to be worked.  (We should not cram a bunch of stuff into "Selected".  We should keep that list short so that it is clear what to work on next.)
+* When adequate capacity is freed up, an Issue is moved from "Selected" to "In Progress" by the developer and it is actively worked.
 * Issues "In Progress" should be worked as quickly as possible to get them done.
 * Once an Issue "In Progress" is ready, it is moved into "In Review" so it can be reviewed by someone.  (The code related to this issue should be submitted on a branch as a pull request to allow the review.)
-* Once the Issue (and associated pull request) is independently reviewed, then the branch is merged and pushed (if the Issue is for a code change) and the Issue is closed.
+* Once the Issue (and associated pull request) is independently reviewed, then the branch is merged and pushed (if the Issue is for a code change) and the Issue may be closed (if the PR completed resolved the issue).
 
-That is the basic Kanban process in a nutshell.
+That is the basic Kanban Issue process in a nutshell.  The process details are given in [TriBITS/CONTRIBUTING.md](https://github.com/TriBITSPub/TriBITS/blob/master/CONTRIBUTING.md).
 
-NOTES:
+## TriBITS Pull Request Process
+
+The Kanban lifecycle process fopr Pull Requests (PRs) follows a reduced number of Kanban stages:
+
+* A new Pull Request is automatically added to the **In Progress** stage.
+* GitHub Actions testing is performed on each push to the PR branch with results displayed on [CDash](https://github.com/TriBITSPub/TriBITS/wiki/TriBITS-CDash-Dashboard).
+* When the PR is ready to be reviewed, it is moved to the **In Review** stage and reviewers are assigned.
+* Once the PR has been reviewed and accepted, it is merged (and the associated topic branch is merged to the main development branch).

```

Comment:
it is merged and moved to the **Done** stage

Reply:
Fixed in #3